ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/PlaylistScriptHandler.swift

```diff
@@ -131,20 +131,8 @@ class PlaylistScriptHandler: NSObject, TabContentScript, TabObserver {
 
     // Copy the item but use the web-view's title and location instead, if available
     // This is due to a iFrames security
-    item = PlaylistInfo(
-      name: item.name,
-      src: item.src,
-      pageSrc: tab.visibleURL?.absoluteString ?? item.pageSrc,
-      pageTitle: tab.title ?? item.pageTitle,
-      mimeType: item.mimeType,
-      duration: item.duration,
-      lastPlayedOffset: 0.0,
-      detected: item.detected,
-      dateAdded: item.dateAdded,
-      tagId: item.tagId,
-      order: item.order,
-      isInvisible: item.isInvisible
-    )
+    item.pageSrc = tab.visibleURL?.absoluteString ?? item.pageSrc
+    item.pageTitle = tab.title ?? item.pageTitle
```

Comment:
`lastPlayedOffset` was reset to 0 before change

Reply:
I had removed it because I didn't see it making any difference in practice, and I didn't want to modify the last played offset for a video which might have updated.

So I made a test, I created a video, then added it to playlist. Then I modified that video to something else on Youtube, and when the video in playlist updated, the offset was wrong. So there is good reason to set it to 0.0 here.

I reverted the change.